app-modules/interaction/tests/Tenant/Filament/Actions/RequestFactories/BulkCreateInteractionActionRequestFactory.php

```diff
@@ -0,0 +1,94 @@
+      in the software, and you may not remove or obscure any functionality in the
+      software that is protected by the license key.
+    - You may not alter, remove, or obscure any licensing, copyright, or other notices
+      of the licensor in the software. Any use of the licensor’s trademarks is subject
+      to applicable law.
+    - Canyon GBS LLC respects the intellectual property rights of others and expects the
+      same in return. Canyon GBS™ and Advising App™ are registered trademarks of
+      Canyon GBS LLC, and we are committed to enforcing and protecting our trademarks
+      vigorously.
+    - The software solution, including services, infrastructure, and code, is offered as a
+      Software as a Service (SaaS) by Canyon GBS LLC.
+    - Use of this software implies agreement to the license terms and conditions as stated
+      in the Elastic License 2.0.
+
+    For more information or inquiries please visit our website at
+    https://www.canyongbs.com or contact us via email at legal@canyongbs.com.
+
+</COPYRIGHT>
+*/
+
+namespace AdvisingApp\Interaction\Tests\Tenant\Filament\Actions\RequestFactories;
+
+use AdvisingApp\CaseManagement\Models\CaseModel;
+use AdvisingApp\Division\Models\Division;
+use AdvisingApp\Interaction\Models\InteractionDriver;
+use AdvisingApp\Interaction\Models\InteractionInitiative;
+use AdvisingApp\Interaction\Models\InteractionOutcome;
+use AdvisingApp\Interaction\Models\InteractionRelation;
+use AdvisingApp\Interaction\Models\InteractionStatus;
+use AdvisingApp\Interaction\Models\InteractionType;
+use AdvisingApp\Prospect\Models\Prospect;
+use AdvisingApp\StudentDataModel\Models\Student;
+use App\Models\User;
+use Exception;
+use Worksome\RequestFactories\RequestFactory;
+
+class BulkCreateInteractionActionRequestFactory extends RequestFactory
+{
+    public function definition(): array
+    {
+        $interactable = $this->faker->randomElement([
+            Student::class,
+            Prospect::class,
+            CaseModel::class,
+        ]);
+
+        throw_unless(in_array($interactable, [
+            Student::class,
+            Prospect::class,
+            CaseModel::class,
+        ], true), new Exception('Invalid interactable type'));
+
+        $interactable = match ($interactable) {
+            Student::class => Student::inRandomOrder()->first() ?? Student::factory()->create(),
+            Prospect::class => Prospect::factory()->create(),
+            CaseModel::class => CaseModel::factory()->create([
+                'case_number' => $this->faker->randomNumber(8),
+            ]),
+        };
+
+        return [
+            'description' => $this->faker->paragraph(),
+            'division_id' => Division::factory(),
+            'end_datetime' => now()->addMinutes(5),
+            'interactable_id' => $interactable->getKey(),
+            'interactable_type' => $interactable->getMorphClass(),
```

Comment:
`interactable_id` and `interactable_type` are not supposed to be here since they are not things submitted to the form of the bulk interaction.

This is not a Model factory, it is supposed to be a factory for the request that would be made to the form.

It's default state should be valid in the form submission.